vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts

```diff
@@ -277,7 +277,7 @@ Please report this issue so it can be remedied or investigated.`), getInstallKey
         const fileErr = new DotnetNoInstallerFileExistsError(new EventBasedError('DotnetNoInstallerFileExistsError',
         `The SDK installation files for version ${specificVersion} running on ${desiredRidPackage} couldn't be found.
 Is the version in support? Note that -preview versions or versions with build numbers aren't yet supported.
-Visit https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core for support information.`), getInstallKeyFromContext(this.context));
+Visit https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core for support information.`), getInstallFromContext(this.context));
```

Comment:
This link should be culture agnostic. Meaning, `https://dotnet.microsoft.com/platform/support/policy/dotnet-core`

Reply:
That's a good catch, thank you. Fixed it!